docs/section22/Rule22-7.md

```diff
@@ -0,0 +1,48 @@
+
+# CHW&CW - Rule 22-7  
+
+**Rule ID:** 22-7  
+**Rule Description:** Baseline chilled water system that does not use purchased chilled water shall be modeled as primary/secondary systems.  
```

Comment:
There is a mismatch in the way the rule is formulated here (as a check that CHW system is modeled with primary/secondary loops) and the RDS logic in lines #26 and down. I would re-formulate the rule in the spreadsheet to focus in verifying primary/secondary arrangement and update the rule logic below accordingly.

Reply:
The original TCD rule was "CHW Loop has a secondary pump", and I changed it to the current description. But the RDS was more written in a way to check that the secondary loop has pump as I'm a bit confused about the focus of this rule. It will be helpful to discuss.